src/Disks/ObjectStorages/AzureBlobStorage/AzureObjectStorage.h

```diff
@@ -113,7 +126,7 @@ class AzureObjectStorage : public IObjectStorage
         const std::string & config_prefix,
         ContextPtr context) override;
 
-    String getObjectsNamespace() const override { return ""; }
+    String getObjectsNamespace() const override { return container ; }
```

Comment:
It seems there is no harm in it but not much good either.

Reply:
ya, I think its good to have it & maintain same structure